code/docs/tests.ss

```diff
@@ -3,26 +3,27 @@
   `((h1 "Running and testing your solutions") (nl)
     (h2 "Overview") (nl)
     "Suppose you're solving "
-    (bold "hello-world")
+    (bold "halting-problem")
```

Comment:
This may or may not be as funny as I think it is... I thought it would be lame if every problem had a `hello-world` description though!

Reply:
Hahaha! I think this is hilarious!